### ISSUE TITLE
[Bug - models and mod compat] Essentials emotes

### ISSUE THREAD

**Issue by alex89912** [Traben-0/Entity_Model_Features#210]
**Describe the bug**
So, I was experimenting with the Fresh Moves resource pack and so I wanted to use an emote, no luck.

**List and link to broken resourcepacks**
[Fresh Moves](https://modrinth.com/resourcepack/tras-fresh-player)

**Expected behavior**

**Screenshots**
https://github.com/Traben-0/Entity_Model_Features/assets/105802970/efc30cdd-ebb3-49b6-afd8-452a80219b85

**(please complete the following information):**
 - Minecraft Version: 1.20.4
 - EMF version 2.0.2
 - ETF version 6.0.1

**Comment by SSJ3Bane** [comment]
Yeah it's just a bug with EMF I believe, not really Fresh Moves, as it's just using the features of EMF. Essentials likely has the same bug due to using similar code or operating where that bug has the same impact.
